components/atom/checkbox/src/styles/settings.scss

```diff
@@ -1,4 +1,5 @@
 $bdrs-atom-checkbox: $bdrs-m !default;
+$bdc-atom-checkbox: rgb(118, 118, 118) !default;
```

Comment:
is there a reason not to use a colour token from `sui-theme` here? 🤔

Reply:
Hmmm I don't know which one to pick exactly, Andres and I decided to put the default one of Chromium. Which one you suggest? 🤔

Reply:
we can have a new one, just keeping it here https://github.com/SUI-Components/sui/blob/master/packages/sui-theme/src/settings/_color.scss